testowy.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
chagne name of your file to index.html - don't use polish language in names of files